pyproject.toml

```diff
@@ -61,8 +61,31 @@ pytest = "^6.2.2"
 pytest-cov = "^2.11.1"
 pytest-mock = "^3.5.1"
 flake8 = "^3.8.4"
-black = {version = "^20.8b1", allow-prereleases = true}
 python-dotenv = "^0.15.0"
+black = {version = "^20.8b1", allow-prereleases = true}
+
+
+[tool.black]
+line-length = 88
+target-version = ['py37']
+include = '\.pyi?$'
+exclude = '''
+
+(
+  /(
+      \.eggs         # exclude a few common directories in the
+    | \.git          # root of the project
+    | \.hg
+    | \.mypy_cache
+    | \.tox
+    | \.venv
+    | _build
+    | buck-out
+    | build
+    | dist
+  )/
+)
```

Comment:
You can remove the directories that aren't relevant to our repo. Build the package with poetry to see what gets generated and should be ignored.

Reply:
To add `black` to our repo I ran: `poetry add --dev --allow-prereleases black`

Running `poetry build` doesn't add the `[tool.black]` section to the `pyproject.toml` file, so I had to add it manually from [here](https://black.readthedocs.io/en/stable/pyproject_toml.html).

Reply:
I think you missed my point. 

This block contains files/folders that will never appear in our folder (like `.hg`). I was suggesting that you build the package using `poetry build` and see which files appear in the folder (like `dist/`) and only include these in this ignore list.

Reply:
I saw that you did do the change that I asked, so ignore my above comment. Looks good!